dir_class.py

```diff
@@ -31,19 +32,22 @@ def get_control_string(self):
     def get_music_files_count(self):
         return len(self.music_files)
 
-    def move(self, destination_subdir, destination_dir):
-        for x, y, z in os.walk(current_dir):
+    def move(self, destination_subdir):
+        destination_dir = "zrobione"
+        for x, y, z in os.walk(self.base_dir):
             destination_pth = destination_dir + os.sep + destination_subdir
             os.makedirs(destination_pth, exist_ok=True)
             for f in z:
                 source_pth = x + os.sep + f
                 shutil.move(source_pth, destination_pth)
-            break
-        shutil.rmtree(current_dir)
 
     def delete(self, base_dir):
         shutil.rmtree(base_dir, ignore_errors=True)
 
     @classmethod
     def parse_control_string(cls, control_string):
         return [s.strip() for s in control_string.split("|")]
+
+    def get_tags(self):
+        for f in self.music_files:
+            audio = EasyID3(f)
```

Comment:
To jest spoko, tylko trzeba to pociągnąć dalej.
Na razie zrób sobie printy, zeby się upewnić, że wiesz jak wyciągać właściwe informacje z obiektów `audio`.
Potem wrócimy do tego na czym właściwie polega issue.